### ISSUE TITLE
Bericht plaatsen op case detailpagina gaat niet goed in Internet Explorer

### ISSUE THREAD

**Issue by MarianSloot** [pietk/werkwel#88]
Tijdens het testen wilde ik in de rol van een expert, ingelogd via Internet Explorer, reageren op een case. De helptekst in het venster waar je kunt typen verdwijnt hier niet als je wilt typen, het is gewone tekst die je kunt selecteren:

![image](https://cloud.githubusercontent.com/assets/13117645/8522124/3dccfce4-23eb-11e5-8da7-8559980b722d.png)

Ik typte een bericht, maar na klikken op de knop 'Reageer' gebeurde er niets. Ik typte het bericht opnieuw, klikt opnieuw op 'Reageer' en weer niets. 
Pas bij een derde keer verscheen het bericht, en dan middenin het scherm:

![image](https://cloud.githubusercontent.com/assets/13117645/8522164/905856e8-23eb-11e5-9f2d-ee92e130b3e5.png)

En wat opvalt: de helptekst is nu weer echte helptekst die verdwijnt als je begint te typen!
Het lijkt alsof de eerste twee berichten onzichtbaar zijn maar wel een zekere ruimte op het scherm innemen.

**Comment by thiscor** [comment]
Die (extra) lege ruimte was me ook al eens opgevallen. Wil je svp bij het testen de eventuele "JavaScript" errors ook bekijken en vermelden in het ticket? Default kun je die errors niet zien, maar bij Chrome met F12 wel en in Internet Explorer moet je expliciet instellen dat je JavaScript errors ziet. Screenshot van de errorconsole toevoegen als er scripterrors zijn.

**Comment by MarianSloot** [comment]
In Firefox heb ik de webontwikkelaars hulpmiddelen ingeschakeld. 
Er zijn foutmeldingen:
![image](https://cloud.githubusercontent.com/assets/13117645/8524218/22cdeb6a-23fa-11e5-8508-4d2c80320437.png)

hopelijk kun je hier iets mee?

**Comment by pietk** [comment]
Even een klein beetje historie van dit probleem:
Eerst werden de reakties van onder naar boven getoond, dus de nieuwste bovenaan. Het invoerveld stond ook bovenaan. Echter, men wilde het andersom, zoals bijv. in WhatsApp: nieuwe berichten onderaan en ook invoer onderaan.
Ik heb dat toen aangepast, en ook al aangegeven dat dit niet werkt in Internet Explorer. Dat was geen probleem, dat werd later wel opgelost...
Als we nu gaan zeggen dat we IE ook supporten, dan kunnen we het beter weer maken zoals het eerst was, lijkt mij.

**Comment by thiscor** [comment]
Iedereen (iig ik) ben gewend van Skype, SMS, whatsapp dat nieuwe berichten onderaan staan en dat je naar boven scrollt als je iets terug wilt
Lezen. Een conversatie lees ik altijd van boven naar beneden, als het andersom is dan vind ik dat verwarrend. Dat et een bug in IE InternetExplorer zit is jammer Microsoft, misschien dat ik er op regenachtige zaterdagavond een keertje naar wil kijken! Geef er maar een lage prioriteit aan. Maar in ieder geval niet weer omdraaien. Ik vind het al lastig genoeg dat al mijn Email programma's de nieuwe berichten bovenaan zetten 😀😀😀😀


Johan Rosman

**Comment by FransClements** [comment]
Met Cor eens! 

Ik ben indertijd voor functionaliteit gegaan. VInd het wel apart een browser belemmerend is voor functionaliteit. Maar het schijnt echt!

Zie: 
![image](https://cloud.githubusercontent.com/assets/12268191/8613995/92806832-26e5-11e5-96d9-20da00944eef.png)

Whatsapp vanaf chrome werkt prima, maar schijnbaar dus niet vanaf explorer. Heb dit indertijd niet beseft! Sorry.

**Comment by tvlastuin** [comment]
Yep, daarom is testen ook zo belangrijk. Prima zoals Marjan dat doet: testen met een goede use-case die in de scope past van de te testen release en met verschillende browsers etc.!!!

Het verdient zelfs aanbeveling om ook te testen met verschillende nog in gebruik zijnde versies van de browsers. Ik heb in het verleden diverse keren meegemaakt dat bij een bedrijf verschillende versies van een browser in gebruik waren en de ene gebruiker dus wel een probleem had en de ander niet.

In ons geval kunnnen we bijvoorbeeld straks meemaken dat we bij Jongeren voor Jongeren heel andere meldingen krijgen wat dit betreft als voor werkgroep45+: die gebruikers weken wellicht nog met oudere Windows versies en oudere versies van verschillende browsers.

Dus eigenlijk zouden we als intern team zoveel mogelijk van dit soort secnario's moeten testen.
In ieder geval weten we dan welke doelgroepen we gaan uitsluiten omdat we bepaalde (versies) van browsers niet ondersteunen. Of andersom geredeneerd: we kunnen van te voren ONZE systeemrequirements duidelijk communiceren.

**Comment by FransClements** [comment]
ah mooi, Johan duwt Marian de code in!

Zet hem op Marian!

Frans

Op 6 juli 2015 14:56 schreef thiscor <notifications@github.com>:

> Die (extra) lege ruimte was me ook al eens opgevallen. Wil je svp bij het
> testen de eventuele "JavaScript" errors ook bekijken en vermelden in het
> ticket? Default kun je die errors niet zien, maar bij Chrome met F12 wel en
> in Internet Explorer moet je expliciet instellen dat je JavaScript errors
> ziet. Screenshot van de errorconsole toevoegen als er scripterrors zijn.
>
>
> > Op 6 jul. 2015 om 14:35 heeft MarianSloot <notifications@github.com>
> het volgende geschreven:
> >
> > Tijdens het testen wilde ik in de rol van een expert, ingelogd via
> Internet Explorer, reageren op een case. De helptekst in het venster waar
> je kunt typen verdwijnt hier niet als je wilt typen, het is gewone tekst
> die je kunt selecteren:
> >
> >
> >
> > Ik typte een bericht, maar na klikken op de knop 'Reageer' gebeurde er
> niets. Ik typte het bericht opnieuw, klikt opnieuw op 'Reageer' en weer
> niets.
> > Pas bij een derde keer verscheen het bericht, en dan middenin het scherm:
> >
> >
> >
> > En wat opvalt: de helptekst is nu weer echte helptekst die verdwijnt als
> je begint te typen!
> > Het lijkt alsof de eerste twee berichten onzichtbaar zijn maar wel een
> zekere ruimte op het scherm innemen.
> >
> > —
> > Reply to this email directly or view it on GitHub.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pietk/werkwel/issues/88#issuecomment-118846942>.
>



-- 
Met vriendelijke groet,

Frans W. Clements
Partner WerkWel
Engelsestadweg 22
3772 TL, Barneveld
Tel.: 06 - 13  75 74 84

**Comment by thiscor** [comment]
@MarianSloot De javascript console die je liet zien bevat alleen maar (oranje) waarschuwingen, die mag je negeren.  Typische javascript error herken je aan de kleur Rood in de error console (Openen met F12 in de meeste browsers)
Chrome:
![chrome2err](https://cloud.githubusercontent.com/assets/11872412/8651437/171f01bc-2979-11e5-922e-4363d2d9dcd2.png)

FireFox:
![fferr](https://cloud.githubusercontent.com/assets/11872412/8651443/1e786606-2979-11e5-921c-8767267f6666.png)

IE:
![ie2err](https://cloud.githubusercontent.com/assets/11872412/8651452/34dab200-2979-11e5-9225-9416ae47e440.png)

of
![ie3err](https://cloud.githubusercontent.com/assets/11872412/8651449/29b015b4-2979-11e5-992f-7b3aecdd901f.png)

**Comment by MarianSloot** [comment]
Ik wil even aangeven, dat het probleem niet alleen in Internet Explorer, maar ook in Firefox optreedt.
Zie issue #91 
@thiscor ik heb je suggestie opgevolgd, en in Firefox de ontwikkelaarshulpmiddelen ingeschakeld. Hoewel er in Firefox geen berichten worden weergegeven (waar Chrome dat wel doet), worden in Firefox geen foutmeldingen gegeven. Een javascript error is dit dus niet.
